### ISSUE TITLE
Cannot read properties of null (reading 'style') with Menu

### ISSUE THREAD

**Issue by withpublic** [grommet/grommet#6163]
I have menu component in a datatable column, e.g.

```
{
property: "attributes",
header: <Text margin="none" size="small">Actions</Text>,
render: (datum) => {
	let items = [
		{
			icon: <Edit/>,
			onClick: () =>
				this.setState({
					showEditForm: true,
					currentLine: datum,
				}),
		},
		{
			icon: <Trash/>,
			onClick: () => this.props.onDelete(datum)
		},
	];

	return (
		<Menu
			icon={<Actions/>}
			items={items}
		/>
	);
},
}
```

When I click on Menu, then on Edit icon, the below error is thrown:

Unhandled Runtime Error
TypeError: Cannot read properties of null (reading 'style')

Call Stack
notifyAlign
node_modules\grommet\es6\components\Drop\DropContainer.js (64:0)
HTMLDocument.place
node_modules\grommet\es6\components\Drop\DropContainer.js (241:0)

### Actual Behavior

No error should be reported

### URL, screen shot, or Codepen exhibiting the issue

<img width="657" alt="image" src="https://user-images.githubusercontent.com/99742304/171574986-6338783b-6e46-4b70-8594-b97cd056e9ee.png">

### Steps to Reproduce
1. Add a datatable;
2. Add Menu into a column;
3. Click a menu item

### Your Environment

"grommet": "^2.23.0",
 "react": "^17.0.2",
"next": "^12.1.5",

Google Chrome
Version 101.0.4951.67 (Official Build) (64-bit)

Windows 11

**Comment by taysea** [comment]
Hi @withpublic, would you be able to provide a codesandbox link that reproduces the issue?

**Comment by jcfilben** [comment]
Closing, unable to reproduce this issue. Feel free to re-open with a codesandbox example that demonstrates the issue.

**Comment by yoisel** [comment]
I got this same issue right now. I cannot share my code, and I was not able to reproduce it in a sandbox, but here is what I know:

1- My project was being migrated from React 17 to React 18. Once I followed the migration steps in here, the error went away:
https://react.dev/blog/2022/03/08/react-18-upgrade-guide#updates-to-client-rendering-apis

 2- Before I fixed the error, I was debugging it for a while, and it seams that this piece of code is at fault:
 https://github.com/grommet/grommet/blob/master/src/js/components/Drop/DropContainer.js#L99
 
```
     useEffect(() => {
      const notifyAlign = () => {
        const styleCurrent = dropRef.current.style;              /// <=== HERE
        const alignControl = styleCurrent.top !== '' ? 'top' : 'bottom';

        onAlign(alignControl);
      };
      // (...) more code
```

At the moment that callback was being invoked, the menu action had already taken place, the menu drop component had already been unmounted, and dropRef.current was nullish. It might be wise to add some null-checks in there.
